app/controllers/admin_controller.rb

```diff
@@ -7,7 +7,11 @@ def index
   def update
```

Comment:
I think we need to lock this down and only allow admins to trigger this action.  Non-admins should not be able to trigger this code.